cmd/agent/dist/conf.d/ntp.d/conf.yaml.default

```diff
@@ -17,3 +17,8 @@ instances:
     # port: 123
     # version: 3
     # timeout: 5
+    #    
+    # Use the ntp servers defined in the host.    
+    # For Unix system, the servers defined in /etc/ntp.conf and etc/xntp.conf are used.
+    # For Windows system, the servers defined in registry key HKLM\SYSTEM\CurrentControlSet\Services\W32Time\Parameters\NtpServer are used.
```

Comment:
this should be reflected in integrations-core

Reply:
let's document as a caveat that enabling this option will make the ntp check unable to detect issues if the clock of the target NTP server of the system is skewed

Reply:
https://github.com/DataDog/integrations-core/pull/4388